api-client/src/test/java/com/xing/api/CallSpecTest.java

```diff
@@ -1206,11 +1211,69 @@ public void canOverrideReadAndConnectTimeoutPerCall() throws Exception {
     }
 
     @Test
-    public void disallowZeroOrNegativeTimeouts() throws Exception {
+    public void canOverrideWriteTimeoutPerCall() throws Exception {
+        // for some unknown reason the timeout is not applied properly when run on Travis CI
+        // so we skip this test on the CI and only execute it locally
+        assumeThat(System.getenv("TRAVIS"), not(equalTo("true")));
```

Comment:
@yayaa There is no other way of testing a timeout like this. Either we do it this way or - I'm afraid - I have to remove this test altogether.